book/src/dev/rfcs/0012-value-pools.md

```diff
@@ -72,38 +74,51 @@ https://zips.z.cash/zip-0209#specification
 
 ### Transparent Transaction Value Pool & Remaining Value
 
-Transparent inputs to a transaction insert value into a transparent transaction value pool associated with the transaction, and transparent outputs remove value from this pool. As in Bitcoin, the remaining value in the pool is available to miners as a fee.
+The sum of unspent *inputs* to the transaction: the *negation* of the sum of the transaction value balances.
```

Comment:
The calculation produces the same value as before, but the definition is much clearer.